proposals/p1964.md

```diff
@@ -314,25 +316,49 @@ states all of Carbon source code should be UTF-8 encoded. Prefix declarations
 would detract the readability of the character literals and increase the
 complexity of character literal [Types](#types).
 
-### Disallowing Numeric Escape Sequences
+### Allowing Numeric Escape Sequences
 
-Not support escaping numeric sequences. This would simplify some of the design
-choices seen above by making types, the restrictions and behavior of character
-literals simpler. Specifically [Types](#types) of `EBCDIC`, by disallowing the
-`\x...` pattern. However this would affect the readability, type-safety and
-consistency of character literals. If we were to disallow numeric escape
+We will not be support escaping numeric sequences. This simplifies some of the
+design choices by making types, the restrictions and behavior of character
+literals simpler. Specifically [Types](#types) of `EBCDIC` by disallowing the
+`\x...` pattern. However this does affect the readability, type-safety and
+consistency of character literals. Since we are disallowing numeric escape
 sequences for character literals, that diverges from the `String` type which
 does allow for numeric escape sequences so we would essentially have the
 functionality to declare character literals but within a string literal.
```

Comment:
```suggestion
This proposal does not support numeric escape sequences using `\x`. This
simplifies the design of character types and literals, making them only
represent code points and not code units. However this does come with the
disadvantage of less consistency of character literals with string literals, since
they now accept different escape sequences. We don't want to remove numeric
escape sequence from string literals, so we can support string use cases like
representing invalid encodings.
```